refinedstorage2-platform-common/src/main/java/com/refinedmods/refinedstorage2/platform/common/integration/recipemod/jei/FilteredResourceGuiContainerHandler.java

```diff
@@ -0,0 +1,54 @@
+package com.refinedmods.refinedstorage2.platform.common.integration.recipemod.jei;
+
+import com.refinedmods.refinedstorage2.platform.api.integration.recipemod.IngredientConverter;
+import com.refinedmods.refinedstorage2.platform.api.resource.filter.FilteredResource;
+import com.refinedmods.refinedstorage2.platform.common.screen.AbstractBaseScreen;
+
+import java.util.Optional;
+import javax.annotation.Nullable;
+
+import mezz.jei.api.gui.handlers.IGuiContainerHandler;
+import mezz.jei.api.ingredients.IIngredientHelper;
+import mezz.jei.api.ingredients.ITypedIngredient;
+import mezz.jei.api.runtime.IClickableIngredient;
+import mezz.jei.api.runtime.IIngredientManager;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+
+public class FilteredResourceGuiContainerHandler implements IGuiContainerHandler<AbstractContainerScreen<?>> {
```

Comment:
Why not use AbstractBaseScreen here instead? That removes the cast on L 33.

```suggestion
public class FilteredResourceGuiContainerHandler implements IGuiContainerHandler<AbstractBaseScreen<?>> {
```

Reply:
good call